docs/[dir]/index.md

```diff
@@ -0,0 +1,31 @@
+# Hello dynamic route
+
+is it working
+
+```js
+display(await FileAttachment("./foo.txt").text());
+```
+
+```js
+display(await FileAttachment("./foo.json").json());
+```
+
+```js
+display(await FileAttachment("./file.json").json());
+```
+
+```js
+import {foo} from "./foo.js";
+
+display(foo);
+```
+
+```js
+display(`${observable.params.dir}.json`);
+```
+
+```js
+display(await FileAttachment("./" + observable.params.dir + ".json").json());
```

Comment:
This is the cool part: you can refer to parameters when constructing `FileAttachment` paths because they’re statically baked into the code!